### DISCUSSION TITLE
基于 Vecel 部署后如何启用 Vecel 下的 analytics 中 Audience的能力

### DISCUSSION THREAD

**Discussion by cigar** [yihong0618/running_page#359] [Q&A] [answered]
我看到Vecel的部署确实比github数据更全面，我也看到了Vecel官方给出了赋能 Analytics 需要添加包的[引导](https://vercel.com/docs/concepts/analytics/audiences/quickstart)，那么我的问题是如果要加一段js，那么我要把这个JS加在项目中哪个主文件呢？谢谢

> Select your framework below to view instructions on using the @vercel/analytics component in your project.

**Comment by yihong0618** [comment]
我不太知道，我感觉 vercel 文档应该有。
@shaonianche 我们找个时间一起研究下？ 

我猜测我们应该属于 others
![image](https://user-images.githubusercontent.com/15976103/211692298-cffcc6e9-b58f-40dd-a96d-136b39fe2ba7.png)

**Reply by shaonianche** [reply]
gatsby 安装插件就可以使用 @vercel/analytics 
https://vercel.com/blog/gatsby-analytics

**Comment by shaonianche** [comment, marked as answer]
> 那么我要把这个JS加在项目中哪个主文件呢？谢谢

添加这两个npm包：gatsby-plugin-vercel、@vercel/analytics，然后参照我的这两个提交修改一下代码就可以了。 https://github.com/shaonianche/running-data-sync/commit/d0ae6c6643fb76abad09ebf6d4c26eb2d9f865a4 、https://github.com/shaonianche/running-data-sync/commit/5cce933efd7ad255ea9bd7edc1b4e53a85aa6acd

**Reply by cigar** [reply]
太效率来。我晚点来试试，谢谢 @shaonianche @yihong0618

**Reply by yihong0618** [reply]
客气，让我们感谢 @shaonianche

**Reply by cigar** [reply]
对照上面的提示，已经成功开启了Audience的能力👍

**Reply by CerealAxis** [reply]
NB
